sample_calendar.json

```diff
@@ -1,78 +1,81 @@
 {
-    "calendars" : {
-        "148460826559529600" : {
-            "calendarState" : 
-            {
-            "startDateString":"20170101",
-            "endDateString":"20170202",
-            "title":"hi",
-            "uniqueId":"148460826559529600",
-            "checkedDays":{}
-            },
+	"writers": {
+		"148460826559529600": {
+			"JULxYGaHuFYlSkXqTC2Fx9KIofp2": true
+		},
+		"148461003056534700": {
+			"JULxYGaHuFYlSkXqTC2Fx9KIofp2": true
+		},
+		"148461051598386460": {
+			"HoQHGYBMwaXAtIKDP5wD1EkjcuX2": true
+		}
 
-             "readers" : {
-                 "JULxYGaHuFYlSkXqTC2Fx9KIofp2" : true
-             },
-             "writers" : {
-                 "JULxYGaHuFYlSkXqTC2Fx9KIofp2" : true
-             }
+	},
+	"readers": {
+		"148460826559529600": {
+			"JULxYGaHuFYlSkXqTC2Fx9KIofp2": true
+		},
+		"148461003056534700": {
+			"JULxYGaHuFYlSkXqTC2Fx9KIofp2": true
+		},
+		"148461051598386460": {
+			"HoQHGYBMwaXAtIKDP5wD1EkjcuX2": true
+		}
+	},
+	"calendars": {
+		"148460826559529600": {
+			"calendarState": {
+				"startDateString": "20170101",
+				"endDateString": "20170202",
+				"title": "hi",
+				"uniqueId": "148460826559529600",
+				"checkedDays": {}
+			}
+		},
 
-         },
-        
-        "148461003056534700" : {
-            "calendarState" :
-            {
-            "startDateString":"20170109",
-            "endDateString":"20170302",
-            "title":"world",
-            "uniqueId":148461003056534700,
-            "checkedDays":{"20170111":1,"20170112":1}
-            },
+		"148461003056534700": {
+			"calendarState": {
+				"startDateString": "20170109",
+				"endDateString": "20170302",
+				"title": "world",
+				"uniqueId": 148461003056534700,
+				"checkedDays": {
+					"20170111": 1,
+					"20170112": 1
+				}
+			}
 
-             "readers" : {
-                 "JULxYGaHuFYlSkXqTC2Fx9KIofp2" : true
-             },
-             "writers" : {
-                 "JULxYGaHuFYlSkXqTC2Fx9KIofp2" : true
-             }
+		},
+		"148461051598386460": {
+			"calendarState": {
+				"startDateString": "20170103",
+				"endDateString": "20170524",
+				"title": "ass",
```

Comment:
Can we title this something other than "ass" now? I open this at work and my calendars are always replaced with "ass" written on the screen in large letters.